### PR TITLE
chore(docs): remove global deployments guide

### PR DESCRIPTION
**Pull request by laktek**
## What kind of change does this PR introduce?

removing global deployments guide for edge function as the information is out of date.

### CODE REVIEW

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -858,6 +857,7 @@ export const functions: NavMenuConstant = {
           url: '/guides/functions/connect-to-postgres',
         },
         { name: 'Testing your Edge Functions', url: '/guides/functions/unit-test' },
+        { name: 'Regional Invocation', url: '/guides/functions/regional-invocation' },
```
Comment (gregnr) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:860]: Double checking this should be here? Does this page exist?

File: apps/www/lib/redirects.js [hunk 2]
```diff
@@ -2318,4 +2318,9 @@ module.exports = [
     source: '/blog/supabase-ai-content-storm',
     destination: 'https://dev.to/supabase/ai-ignites-the-rain-content-storm-is-back-kdl',
   },
+  {
+    permanent: true,
+    source: '/docs/guides/functions/global-deployments',
+    destination: 'docs/guides/functions',
+  },
```
Comment (gregnr) [inline apps/www/lib/redirects.js:2325]: Just noticed you also have this PR https://github.com/supabase/supabase/pull/18617/files

Perhaps we let that one merge first, and then set this redirect to `/guides/functions/regional-invocation`? Seems somewhat closely related.

```suggestion
  {
    permanent: true,
    source: '/docs/guides/functions/global-deployments',
    destination: 'docs/guides/functions/regional-invocation',
  },
```

Comment (laktek) [reply apps/www/lib/redirects.js:2325]: yeah, sounds good.

File: apps/www/lib/redirects.js [hunk 3]
```diff
@@ -2318,4 +2318,9 @@ module.exports = [
     source: '/blog/supabase-ai-content-storm',
     destination: 'https://dev.to/supabase/ai-ignites-the-rain-content-storm-is-back-kdl',
   },
+  {
+    permanent: true,
+    source: '/docs/guides/functions/global-deployments',
+    destination: 'docs/guides/functions/regional-invocation',
```
Comment (gregnr) [inline apps/www/lib/redirects.js:2324]:
```suggestion
    destination: '/docs/guides/functions/regional-invocation',
```

**Comment by laktek** [comment]
@gregnr Good to merge this? #18617 was merged

**Review by gregnr — APPROVED** [review]

**Comment by gregnr** [comment]
Looks like we need a `www` coder owner review cc @supabase/website

**Review by alaister — APPROVED** [review]

**Pull request merged**